saddle-core/src/main/scala/org/saddle/scalar/ScalarTagBool.scala

```diff
@@ -57,4 +57,12 @@ object ScalarTagBool extends ScalarTag[Boolean] {
   def parse(s: String) = s.toBoolean
 
   override def toString = "ScalarTagBool"
+
+  override def copySlice(
+      from: Int,
+      to: Int,
+      source: Array[Boolean]
+  ): Array[Boolean] = {
+    java.util.Arrays.copyOfRange(source, from, to)
```

Comment:
Did you check whether the original generic specialized `copySlice` calls the hand-specialized `copyOfRange` [overloads](https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#copyOfRange(boolean[],%20int,%20int)) or whether it calls the generic `copyeOfRange[T]`?
In the first case, it wouldn't be necessary to move the method in ScalarTag. That'd be nice. I find ScalarTag a bit bloated.